client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -9,30 +10,38 @@ import CardHeading from 'calypso/components/card-heading';
 import Notice from 'calypso/components/notice';
 import NoticeAction from 'calypso/components/notice/notice-action';
 import { IntervalLength } from 'calypso/my-sites/marketplace/components/billing-interval-switcher/constants';
-import { isEligibleForProPlan } from 'calypso/my-sites/plans-comparison';
 import { getBillingInterval } from 'calypso/state/marketplace/billing-interval/selectors';
-import { getSelectedSite } from 'calypso/state/ui/selectors';
 import { isAtomicSiteWithoutBusinessPlan } from './utils';
 
 // Mapping eligibility holds to messages that will be shown to the user
 function getHoldMessages(
 	context: string | null,
 	translate: LocalizeProps[ 'translate' ],
-	eligibleForProPlan: boolean,
-	billingPeriod?: string
+	billingPeriod?: string,
+	isMarketplace?: boolean
 ) {
 	return {
 		NO_BUSINESS_PLAN: {
-			title: eligibleForProPlan
-				? translate( 'Upgrade to a Pro plan' )
-				: translate( 'Upgrade to a Business plan' ),
+			title: ( function () {
+				if ( isMarketplace && isEnabled( 'marketplace-personal-premium' ) ) {
+					return translate( 'Upgrade to a Personal plan' );
+				}
+
+				return translate( 'Upgrade to a Business plan' );
+			} )(),
```

Comment:
It would still depend on the `plans/pro-plan` feature flag being `true` which I don't anticipate happening. Also, if we fallback again to these plans, `starter` plan should be advertised here in which we have already launched the marketplace. So, yeah, if a site is free they will need to upgrade to `business` or `starter` if the feature flag is enabled and we make the changes that are lost probably due to the plans rollback.